website/docs/blueprints/data-analytics/binpacking-custom-scheduler-eks.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 6
+sidebar_label: Binpacking for Amazon EKS
+---
+
+
+# Binpacking for Amazon EKS
+
+## Introduction
+By default, the [scheduling-plugin](https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins) NodeResourcesFit use the ```LeastAllocated``` for score strategies. For the long running workloads, that is good because of high availability. But for batch jobs, like Spark workloads, this would have high cost. By changing the from ```LeastAllocated``` to ```MostAllocated```, it avoids spreading pods across all running nodes, leading to higher resource utilization and better cost efficiency. In this post, we will show you how to enable binpacking with Amazon EKS.
+
+
+## Deploying the Solution
+Please follow [this link](https://github.com/aws-samples/custom-scheduler-eks) to deploy the custom scheduler.
+
+
+## Set up pod template to use the custom scheduler for Spark
+We should add custom scheduler name to the pod template as follows
+```bash
+kind: Pod
+spec:
+  schedulerName: my-scheduler
```

Comment:
Let's give the name as `custom-k8s-scheduler`

Reply:
Neuron scheduler deployment is also using the same name :)

Reply:
done